xrpl/deploy-multisig.js

```diff
@@ -0,0 +1,112 @@
+const xrpl = require('xrpl');
+const { Command, Option } = require('commander');
+const { mainProcessor, getWallet, getAccountInfo, sendTransaction, hex } = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const { printInfo, printWarn } = require('../common');
+
+const MAX_TICKET_COUNT = 250;
+const KEY_TYPE = xrpl.ECDSA.secp256k1;
+
+const BASE_RESERVE_XRP = 10;
+const OWNER_RESERVE_XRP = 0.2;
+
+async function processCommand(_, chain, options) {
+    const client = new xrpl.Client(chain.rpc);
+    await client.connect();
+
+    const wallet = await getWallet(chain, options);
+    const balance = Number((await getAccountInfo(client, wallet.address)).Balance) / 1e6;
+    const multisigReserve = BASE_RESERVE_XRP + MAX_TICKET_COUNT * OWNER_RESERVE_XRP;
+    if (balance < Number(multisigReserve)) {
+        printWarn(`Wallet XRP balance is insufficient to fund the multisig account reserve (${multisigReserve} XRP)`);
+        process.exit(0);
+    }
+
+    printInfo('Creating multisig account');
+    const multisig = xrpl.Wallet.generate(KEY_TYPE);
```

Comment:
```suggestion
    const multisig = xrpl.Wallet.fromSeed(...);
```
can the initial multisig be generated from the seed? This also allows creating the same multisig address between testnet and mainnet which is convenient (similar to how we deploy EVM gateways to the same address)

Reply:
Added this in latest commit: https://github.com/axelarnetwork/axelar-contract-deployments/pull/564/commits/dedd8d8f5f8df582f2507b27f854197d49df4c9f. The deploy script now converts the env wallet into the XRPL multisig account, unless the `--generate-wallet` flag is used.